hail/testng-cpp-codegen.xml

```diff
@@ -0,0 +1,11 @@
+<suite name="CppCodegenSuite" verbose="1">
+    <test name="TestCppCodegen">
+      <classes>
+           <class name="is.hail.nativecode.NativeCodeSuite"/>
+           <method name="is.hail.annotations.AnnotationsSuite.testReadWrite"/>
```

Comment:
I don't see where the environment variable to enable cpp codegen is set. I thought I had added that to pom.xml

Reply:
Oh, good catch!  I removed the tests from the pom.  I now set the flag in the cpp tests in the build.yaml.